lessons/2.writing-our-first-program.md

```diff
@@ -3,8 +3,54 @@ Writing Our First Program
 
 In this lesson, we'll examine one of the simplest programs in V.
 
+Yes it is the iconic [Hello World](https://en.wikipedia.org/wiki/%22Hello%2C_World!%22_program)
+that is often the first program someone would write in language they are learning.
+
 ```v
 fn main() {
-	println('hello world')
+	println('Hello, World!')
 }
 ```
+
+In V, functions are declared with the `fn` keyword, followed by the name of the
+function (in this case `main`), the parameters to the function, if any (in this
+case there aren't any), then an opening curly brace, `{`, on the same line.
+
+V also has 2 ways to output text - one that only prints what is given to it,
```

Comment:
If this for absolute beginners maybe mention *where* the output can be seen? (A terminal/shell??)

Reply:
Hmm... that gets tricky, as you may have the output redirected to a file, etc.  Let me see...